.github/workflows/macos.yml

```diff
@@ -79,7 +79,9 @@ jobs:
         env:
           SCONS_CACHE: ${{github.workspace}}/.scons_cache/
         run: |
-          scons -j2 verbose=yes warnings=extra werror=yes platform=macos tests=no target=${{matrix.target}} dev_build=no
+          scons -j$(sysctl -n hw.ncpu) arch=x86_64 verbose=yes warnings=extra werror=yes platform=macos tests=no target=${{matrix.target}} dev_build=no
```

Comment:
It isn't necessary to specify `-j` if you want all cores to be used, SCons will auto-detect them